.github/workflows/test_scala_fmt.yaml

```
@@ -32,6 +41,14 @@ jobs:
           distribution: 'temurin'
           java-version: '11'
 
+      - name: Install curl
+        run: |
+          apt-get update && apt-get install -y curl
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)